Biomit_workshop_3_Reporting with R markdown.Rmd

```diff
@@ -342,11 +342,13 @@ Here is where it get fancy....In the YAML header you can specify the custom para
 
 library(dplyr)
 reproducible_tab <- NHANES %>% filter(Education == "params$education") %>% 
```

Comment:
params$education is "Some College" by default which is a string name of the factor variable. I understand that it is working inside dplyr chain commands without any additional syntax because it is not a column name yes?